toolbox/githubctl/main.go

```diff
@@ -24,29 +24,28 @@ import (
 )
 
 var (
-	owner                = flag.String("owner", "istio", "Github owner or org")
-	tokenFile            = flag.String("token_file", "", "File containing Github API Access Token")
-	op                   = flag.String("op", "", "Operation to be performed")
-	repo                 = flag.String("repo", "", "Repository to which op is applied")
-	baseBranch           = flag.String("base_branch", "", "Branch to which op is applied")
-	refSHA               = flag.String("ref_sha", "", "Reference commit SHA used to update base branch")
-	nextRelease          = flag.String("next_release", "", "Tag of the next release")
-	updateDownloadScript = flag.Bool("update_download_script", false, "Have download script point to latest version")
-	githubClnt           *u.GithubClient
+	owner       = flag.String("owner", "istio", "Github owner or org")
+	tokenFile   = flag.String("token_file", "", "File containing Github API Access Token")
+	op          = flag.String("op", "", "Operation to be performed")
+	repo        = flag.String("repo", "", "Repository to which op is applied")
+	baseBranch  = flag.String("base_branch", "", "Branch to which op is applied")
+	refSHA      = flag.String("ref_sha", "", "Reference commit SHA used to update base branch")
+	nextRelease = flag.String("next_release", "", "Tag of the next release")
+	githubClnt  *u.GithubClient
 )
 
 const (
 	istioVersionFile     = "istio.VERSION"
 	istioDepsFile        = "istio.deps"
 	releaseTagFile       = "istio.RELEASE"
-	downloadScript       = "downloadIstio.sh"
+	downloadScript       = "downloadIstioCandidate.sh"
```

Comment:
don't you need release/downloadIstioCandidate.sh ?
(I put the new script under release to limit the chance of accidental changes to it - we can't move the other one as the URL is burned)

Reply:
You are right. I didn't notice the two download scripts are in different directories.